### PR TITLE
Update zhtr.js (v4.2.2)

### PR DESCRIPTION
**Pull request by 0v0katai**
Updated to v4.2.2

### CODE REVIEW

File: zhtr.js [hunk 1]
```diff
@@ -89,12 +89,12 @@ settings.local.region 預設伺服器地區
 settings.local.lang 語言
 settings.network.header 網路
 settings.network.netRerouting 動態路由（影響網路時延）
-settings.network.netRate Network Rate (Hz)
+settings.network.netRate 網路傳送率
```
Comment (mty22) [inline zhtr.js:92]: need to keep `(Hz)`

**Review by mty22 — CHANGES_REQUESTED** [review]

**Review by Awesome-Boy-001 — COMMENTED** [review]
Looks great! Sentences and wordings are also very professional and specific.

**Review by Awesome-Boy-001 — COMMENTED** [review]
Words and sentences are now much better and smooth. Didn't see or catch mistakes.

**Review by Awesome-Boy-001 — COMMENTED** [review]
Looks good. Didn't find mistakes in the latest commit.

**Pull request merged**